### ISSUE TITLE
Add detachEphemeralDisk service to virtual guest interface

### ISSUE THREAD

**Issue by xingzhou** [maximilien/softlayer-go#22]

**Comment by maximilien** [comment]
Is this done now? Or maybe need to move to the CPI project?

**Comment by maximilien** [comment]
Let's make sure to add a description for this. If done then let's close it. Thanks.

**Comment by maximilien** [comment]
Hi, @xingzhou,

Can you please update us on this issue. If it's done, let's close it.

Thanks,

@maximilien and @posijon

**Comment by maximilien** [comment]
@xingzhou is there a Tracker story for this? I am trying to clean up things. Can you update me?

Thanks,

@maximilien

**Comment by maximilien** [comment]
@xingzhou still waiting on this. Can you talk to @jianqiu and see if he wants to inherit this.

/cc @zhang-hua

**Comment by maximilien** [comment]
Hi, @xingzhou and @jianqiu , 

Doing some cleanup. Do we have resolution on this? 

I am planning to close, unless I hear otherwise within 24 hours.

Thanks,

@maximilien

/cc: @zhang-hua and @posijon

**Comment by jianqiu** [comment]
Hi, @maximilien 

I am now looking into this feature,  based on my knowledge, I think it is a valuable feature to implement, since we had enabled ordering ephemeral disk during `creat_vm`, it is necessary to revoke it when doing `delete_vm` in case softlayer continues to charge money. I will create a new story in Tracker tomorrow.

Thanks!

**Comment by maximilien** [comment]
Hi, @jianqiu,

Ok, great. Let's make sure to link the tracker story to this and once it's solved comment here so I close this.

Thanks much for paying attention to this. Best,

@maximilien

**Comment by maximilien** [comment]
@jianqiu,

Can you add the story number here. I will close this issue once that tracker story is accepted.

Thanks,

max

**Comment by maximilien** [comment]
Closing this based on innactivity
